ontology/GALLONT.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: ontology_detail
+id: gallont
+title: "Plant Gall Ontology"
+contact:
+  email: adeans@psu.edu
+  github: adeans
+  label: Andy Deans
+  orcid: 0000-0002-2119-4663
+dependencies:
+  - id: pato
+  - id: po
+  - id: ro
+  - id: flopo
+  - id: caro
+  - id: obi
+  - id: poro
+  - id: ncbitaxon
+description: Ontology of plant gall phenotypes. Plant galls are novel plant structures, generated by plants in response to biotic stressors. This ontology is used to annotate gall phenotypes (e.g., their colors, textures, sizes, locations on the plant) in a semantic way, in order to facilitate discoveries about the genetic and physiologic mechanisms responsible for such phenotypes. The ontology can also be used as a controlled vocabulary for natural language descriptions of plant galls.
+domain: phenotype
+homepage: https://github.com/adeans/gallont/tree/main
+license:
+  url: https://creativecommons.org/publicdomain/zero/1.0/
+  label: CC0 1.0
+preferredPrefix: GALLONT
+products:
+  - id: gallont.owl
```

Comment:
missing OBO product, would also be nice to configure JSON output

Reply:
@cthoyt I will add OBO. How do I get it to produce json?

Reply:
That's an ODK configuration question for @anitacaron :)

Reply:
Hi @adeans, you can just add json in the list of export_formats in the odk config file. For example: 

https://github.com/obophenotype/cell-ontology/blob/2180772b156cf79797b04cc494efc8185088428c/src/ontology/cl-odk.yaml#L10-L13

Reply:
Thanks, @anitacaron . I am [generating a json](https://github.com/adeans/gallont/blob/main/gallont.json) version now. I also changed the homepage to https://adeans.github.io/gallont/ which I hope to flesh out more like PATO's: https://pato-ontology.github.io/pato/. Should I update gallont.md?

Reply:
AWESOME @adeans yes just use "suggest mode" and update the md file right in the PR!

Reply:
<img width="512" alt="image" src="https://github.com/OBOFoundry/OBOFoundry.github.io/assets/7070631/ed8ff699-a3cd-4287-a26f-769461e56070">